assets/js/blocks/cart-checkout/checkout-i2/form-step/additional-fields.tsx

```diff
@@ -1,25 +1,36 @@
 /**
  * External dependencies
  */
-import { InnerBlocks } from '@wordpress/block-editor';
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
 import {
 	RegisteredBlocks,
 	getRegisteredBlocks,
 } from '@woocommerce/blocks-checkout';
+import { useRef } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import './editor.scss';
+import { useForcedLayout } from '../use-forced-layout';
 
 export const AdditionalFields = ( {
 	area,
 }: {
 	area: keyof RegisteredBlocks;
 } ): JSX.Element => {
+	const registeredBlocks = getRegisteredBlocks( area );
+	const { 'data-block': clientId } = useBlockProps();
```

Comment:
`useBlockProps` exposes `clientId` in the `data-block` attribute, we're pulling that value here. Other option including prop drilling it from above in the Edit function.
`useBLockProps` data is meant to be spread over a div, so it's not named `clientId`.

Reply:
I guess my question is more, why `data-block` and not `data-clienId`

Reply:
Well, that's what's returned by `useBlockProps` I'm renaming `data-block` to `clientId`, not the other way around. Or do you want me to use `data-client-id` instead of `clientId`?

Reply:
ah I understand now. Sorry, I misread. Leave it alone 👍🏻